packages/admin/src/pages/super-admin-dashboard/user/[id]/index.page.tsx

```diff
@@ -16,15 +19,26 @@ export const getServerSideProps = async ({
 }: GetServerSidePropsContext) => {
   const userToken = getUserTokenFromCookies(req);
 
-  const getPageData = () => getUserById(params?.id as string, userToken);
+  const getPageData = async () => {
+    const jwtUser = await getUserFromJwt(userToken);
+    const isViewingOwnAccount = jwtUser?.gapUserId === params?.id;
+
+    return {
+      isViewingOwnAccount,
+      ...(isViewingOwnAccount
+        ? jwtUser
+        : await getUserById(params?.id as string, userToken)),
+    };
```

Comment:
Find this a bit more readable
```suggestion
    const user = isViewingOwnAccount
        ? jwtUser
        : await getUserById(params?.id as string, userToken);
    return {
      isViewingOwnAccount,
      ...user,
    };
```

Reply:
Fine with me, added that